testing/test-config.js

```diff
@@ -91,14 +90,7 @@ export class TestConfig {
   }
 
   skipIfPropertiesObfuscated() {
-    return this.skip(function () {
-      return window.__karma__.config.amp.propertiesObfuscated;
-    });
-  }
-
-  enableIe() {
-    this.skipMatchers.splice(this.skipMatchers.indexOf(this.runOnIe), 1);
-    return this;
+    return this.skip(() => window.__karma__.config.amp.propertiesObfuscated);
```

Comment:
The inner function does not need access to the bound `this` reference, so could be either an arrow or standard function. Might roll this change back to avoid additional complication.